core/src/main/resources/lib/form/hetero-list.jelly

```diff
@@ -84,11 +84,11 @@ THE SOFTWARE.
         <j:set var="help" value="${descriptor.helpFile}" />
         <j:if test="${hasHeader}">
           <div class="tr help-sibling">
-            <div colspan="3">
-              <div class="${attrs.disableDragAndDrop or readOnlyMode ? '' : 'dd-handle'}">
+            <div colspan="3" style="display: flex;">
```

Comment:
This should probably be extracted to a CSS class, maybe above https://github.com/jenkinsci/jenkins/blob/fa1fa7e7e1ccc3ad4e870d78a05307660b48d0f8/war/src/main/less/base/style.less#L1283

A possible selector could be `.repeated-container .heading`.

Reply:
I test it in the _Global Tool Configuration_ page, everything looks fine.
Besides, I've move the style to the CSS(less) file, have a look a it.